libfwupd/fwupd-request.h

```diff
@@ -53,12 +54,33 @@ typedef enum {
 /**
  * FWUPD_REQUEST_ID_PRESS_UNLOCK:
  *
- * The user needs to press unlock on the device.
+ * The user needs to press unlock on the device to continue, e.g.
+ * "Press unlock on the device to continue the update process."
  *
  * Since 1.6.2
  */
 #define FWUPD_REQUEST_ID_PRESS_UNLOCK "org.freedesktop.fwupd.request.press-unlock"
 
+/**
+ * FWUPD_REQUEST_ID_REMOVE_DEVICE:
```

Comment:
Maybe make this `REQUEST_ID_REMOVE_USB_DEVICE` or change the string to drop the word USB.
Otherwise we have a problem some day with some other sort of PCIe device or such.

Reply:
That's good

Reply:
Fixed.